<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoDialog</class>
 <widget class="QDialog" name="DemoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="helloPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="helloLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="helloSpinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="helloLcdNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
